README.md

```diff
@@ -62,6 +62,9 @@ Fields:
  - `enabledFeatures`. Type: \[String\]. Optional. Default value: empty array. Used on the front end to determine which opt-in features the institution has access to. (Either make this required, or make `disabledFeatues` optional. If this field is left optional, remove default value.)
  - `facebookId`: Type: String. Optional. The id of the facebook page using our facebook app that is associated with the institution.
  - `filterOn` Type: Boolean. Optional. The field that distinguished open bots (value `false`) from closed bots (value `true`). (To do: move this to `dialogSettings`. Make required.)
+ - `freeze` Type: Object. Optional. Enable and control "freezing" the bot when an admin sends a message.  See AH-1054
+   - `enable` Type: Boolean.  If true, enable freezing.
+   - `seconds` Type: Integer.  Number of seconds to freeze the bot for.
```

Comment:
We'll have two time fields; one for how long the window is after an admin response, and how long to delay messages (e.g., give a four-hour window during which messages are frozen, but if there's no response for 20 minutes, assume the admin stepped away and deliver a Holocene response.)

Maybe something around "window" or "thawTimeout"?  (Though it's not clear if we thaw the user or individual messages, so that's ambiguous.)

Reply:
That was future tense.  However, given that I'm already started on the second one, I'll go ahead and add that in this PR to avoid having to do another round of waiting on admithub-common.

Reply:
I added the other field and renamed them.